prow/jobs.yaml

```diff
@@ -67,6 +67,13 @@ kubernetes/kubernetes:
       # Bazel needs privileged mode in order to sandbox builds.
       securityContext:
         privileged: true
+      resources:
+        requests:
+          cpu: 6
+          memory: 10G
+        limits:
+          cpu: 6
```

Comment:
I wonder if 6 needs to be quoted. Also Gi is the binary SI version G is decimal. Probably doesn't matter